client/base_client.go

```diff
@@ -328,9 +328,10 @@ func (c *baseClient) getMembers(ctx context.Context, url string, timeout time.Du
 	if err != nil {
 		return nil, err
 	}
-	members, err := pdpb.NewPDClient(cc).GetMembers(ctx, &pdpb.GetMembersRequest{})
-	if err != nil {
-		attachErr := errors.Errorf("error:%s target:%s status:%s", err, cc.Target(), cc.GetState().String())
+	members, _ := pdpb.NewPDClient(cc).GetMembers(ctx, &pdpb.GetMembersRequest{})
```

Comment:
I think we can not just ignore the error here. The error inside header and the gRPC error should be returned both.

Reply:
> I think we can not just ignore the error here. The error inside header and the gRPC error should be returned both.

I do this because the function GetMembers does not return an error.